### ISSUE TITLE
Add a task to remove app_dev.php

### ISSUE THREAD

**Issue by armetiz** [TheBigBrainsCompany/capistrano-symfony#7]
I think that app_dev.php have to be removed from production or foo-environment,
So, I propose to create a new task, and this task can be added by user only after some specific hook.

@borisguery, what do you think ?

Thomas.

**Comment by borisguery** [comment]
I completely missed this issue ^^

I did a simple reverse removal according to the defined symfony environment : https://github.com/TheBigBrainsCompany/capistrano-symfony/commit/a94a3d4a014aeb0d314ce674409e54395cb5c9e2

**Comment by armetiz** [comment]
So cool !

**Comment by armetiz** [comment]
An option should be used to disable this feature for debuging purpose.

**Comment by borisguery** [comment]
Capifony use an option named `clear_controllers`. We may add a similar option to enable/disable on demand with a default on enable.

**Comment by armetiz** [comment]
It should be a great choice. I'll add this ASAP.

**Comment by carlescliment** [comment]
I have some specific app_xxx.php I **do** want to deploy. Is there any way to override it? Something like:

``` ruby
set :controllers_to_clear, ["app_dev.php", "app_test.php"]
```
